config/env/default.js

```diff
@@ -42,6 +42,11 @@ module.exports = {
   },
   logo: 'modules/core/client/img/brand/logo.png',
   favicon: 'modules/core/client/img/brand/favicon.ico',
+  illegalUsernames: ['meanjs', 'fwd', 'fwd:', 'reply', 'administrator', 'password', 'community',
+                   'unknown', 'anonymous', 'null', 'undefined', 'home', 'signup', 'signin', 'login',
+                   'edit', 'settings', 'demo', 'support', 'networks', 'profile', 'avatar', 'mini',
+                   'photo', 'account', 'api', 'modify', 'feedback', 'security', 'accounts', 'tribe', 'tag'
```

Comment:
In fact I recommend getting rid of lots of these, such as avatar, demo, unknown etc... it should be up to the user to populate this list and MEAN should only reserve very important ones like admin out the box IMO.